recipes/dialign-tx/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: dialign-tx 
+  version: '1.0.2'
+
+source:
+  fn: DIALIGN-TX_1.0.2.tar.gz
+  url: http://dialign-tx.gobics.de/DIALIGN-TX_1.0.2.tar.gz 
+  md5: 8ccfb1d91136157324d1e513f184ca29
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc  # [not osx]
+    - zlib
```

Comment:
please add `{{CONDA_ZLIB}}*` to pin zlib, also in the run section :)

Reply:
I actually think i need zlib, leftover from a skeleton meta file. Still builds fine on my local system without it.